### PR TITLE
Sample Data

### PR DESCRIPTION
**Pull request by Pratik-Karnik**
#4

### CODE REVIEW

File: sampleShopcarts.csv [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-1005,2008,"Roscatto Wine",12,23
-1006,2007,"Spa Sensation Mattress",150,6
-1007,2006,"Bathroom Curtain",1.99,1
-1008,2005,"Oven Tray",4.99,11
-1009,2004,"Columbia Boots",90,1
-1010,2003,"Fila Running Shoes",20,4
-1011,2002,"5 piece bedroom set",42,3
-1012,2001,"Macbook Pro",1700,1
\ No newline at end of file
+uid,sid,products__sku,products__quantity,products__name,products__unitprice
```
Comment (senongtor) [inline sampleShopcarts.csv]: I think it's better to only have one underscore..?

Comment (Pratik-Karnik) [reply sampleShopcarts.csv]: Didnt realize there were two. Lemme change it

Comment (Pratik-Karnik) [reply sampleShopcarts.csv]: Done

File: sampleShopcarts.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-uid,sid,products__sku,products__quantity,products__name,products__unitprice
+uid,sid,products_sku,products_quantity,products_name,products_unitprice
 1,1,123456780,2,"Settlers of Catan",27.99
 1,1,876543210,1,"Risk",27.99
 2,2,,,
```
Comment (senongtor) [inline sampleShopcarts.csv]: Line 4 missing a comma

**Pull request merged**